src/bin/agent/CommandLockAll.ts

```diff
@@ -27,7 +26,7 @@ class CommandLockAll extends CommandPolykey {
         this.fs,
         this.logger.getChild(binProcessors.processClientOptions.name),
       );
-      const meta = await binProcessors.processAuthentication(
+      const auth = await binProcessors.processAuthentication(
```

Comment:
What is the type of this `auth`?

Reply:
It used to be a GRPC metadata object. Now it's `{ Authorization?: string }`. I can use it as the `metadata` for the message, or I can combine it with other metadata.

```ts
{
  metadata: auth,
  something: 123,
}
// OR
{
  metadata: {
  some: 'data',
   ...auth,
  }
  some: 'param',
}
```

Reply:
I think you should use `token` or something similar to how http does for their headers. At the very least should all be lowercase.

Reply:
I'll make it lowercase `authorization?: string`.  It can be a token or the raw password so `authorization` is more appropriate than `token`.

Reply:
Fixed.